doc/api/repl.md

```diff
@@ -321,17 +326,17 @@ The following example shows two new commands added to the REPL instance:
 ```js
 const repl = require('repl');
 
-var replServer = repl.start({prompt: '> '});
+const replServer = repl.start({prompt: '> '});
 replServer.defineCommand('sayhello', {
   help: 'Say hello',
-  action: function(name) {
+  action: function sayHello(name) {
```

Comment:
I understand. Also I do not have strong feelings about this, but in the `/lib` case it would hae value since stack traces might look nicer. Here showing APIs is the prime thing to do. E.g. you will never see full error handling in programming books. Again. I don't mind that's why I left it as a comment.